### PR TITLE
Miscellaneous coroutine tagtype fixes

### PR DESCRIPTION
**Pull request by SongRe**
Things I missed in my original PR [here](https://github.com/FlyingsMarmot/llvm-project/pull/1)

### CODE REVIEW

**Comment by abdurj** [comment]
hm, ur right this can go first

**Comment by abdurj** [comment]
can you undo these changes while you're at it

https://github.com/FlyingsMarmot/llvm-project/pull/1/files#diff-450bf1d395e6da1ee7aa09b6216d7c35021a2ee6ae0ff613bfe6f53f0c29d663R1823-R1825

I think this is what broke the prior clang test

**Comment by SongRe** [comment]
> can you undo these changes while you're at it
> 
> https://github.com/FlyingsMarmot/llvm-project/pull/1/files#diff-450bf1d395e6da1ee7aa09b6216d7c35021a2ee6ae0ff613bfe6f53f0c29d663R1823-R1825
> 
> I think this is what broke the prior clang test

Fixed

File: clang/lib/Sema/SemaCodeComplete.cpp [hunk 1]
```diff
@@ -1822,10 +1822,6 @@ static void AddTypeSpecifierResults(const LangOptions &LangOpts,
     Results.AddResult(Result("_Coroutine", CCP_Type));
     Results.AddResult(Result("wchar_t", CCP_Type));
 
-    Builder.AddTypedTextChunk("_Coroutine");
```
Comment (abdurj) [inline clang/lib/Sema/SemaCodeComplete.cpp:1825]: xd if we ever get time maybe we can figure out how to do this properly

**Review by abdurj — APPROVED** [review]

**Pull request merged**